rn/Teacher/test/scripts/jestSetup.js

```diff
@@ -130,7 +130,7 @@ NativeModules.RNSound = {
 }
 
 NativeModules.NativeNotificationCenter = {
-  postAsyncActionNotification: jest.fn(),
```

Comment:
I think this is used in our async actions, right? We have `CoreDataSyncHelper.swift` that listens to specific async actions and it's important for things like making sure legacy courses get synced to core data. Just want to make sure all that stuff isn't broken.

Reply:
`CoreDataSync.syncAction` is called, not `postAsyncActionNotification`, which was not defined in the `NativeNotificationCenter`.